src/components/DocsContent/index.js

```diff
@@ -138,7 +138,7 @@ const mdxGrommetMap = {
     else
       return (
         <Grommet.Box pad="large">
-          <Grommet.Image {...props} />
+          <Grommet.Image {...props} width="100%" />
```

Comment:
Can't we just not resize the images at all and leave them at whatever size they normally would be?

Reply:
this will lead to very weird resizing by default.

Reply:
This fixes the issues and anyways this is the width, so it won't cause any problem. Can you check https://centrifuge-docs.netlify.app?